doc/api/globals.md

```diff
@@ -81,6 +81,18 @@ This variable may appear to be global but is not. See [`exports`].
 ## global
 <!-- YAML
 added: v0.1.27
+deprecated: REPLACEME
+-->
+
+<!-- type=global -->
+
+> Stability: 0 - Deprecated: Use [`globalThis)`][] instead.
+
+* {Object} The global namespace object.
+
+## globalThis
```

Comment:
I'm not sure we should document this as it's a JavaScript global.

Everything else in globals.md is a Node.js specific global that is not available in "plain" JavaScript. That is - nothing else here is defined in the ECMAScript spec (we don't list Array, String etc here for example).

Reply:
well it's currently not in ecma262, it's still stage 3

Reply:
I would suggest holding off for the time being